### DISCUSSION TITLE
How can I create new Users and assign permissions

### DISCUSSION THREAD

**Discussion by LudeeD** [inventree/InvenTree#4387] [Q&A] [answered]
I've installed InvenTree with Docker and I can login as the root user that I created.

From here how can I create new users and assign them permissions ? 

I cannot see anything in the settings part and also if I access the django admin page I can create a user but then how can I assign them roles like the ones you talk about in the documentation? Do I need to create these myself ?

I honestly read the documentation but I cannot find the answer for this :(

**Comment by eeintech** [comment, marked as answer]
Yes you need to create roles yourself: https://docs.inventree.org/en/latest/settings/permissions/#roles
Then you can add users to them.

You can even set the default role new users are assigned to, see "Group on signup":
![image](https://user-images.githubusercontent.com/4020546/220447476-fb97d0fd-5fde-437d-b49b-170a906da9b2.png)

**Comment by LudeeD** [comment]
Ah got it. When I create the group will allow me to specify each individual rule. Thanks for the quick help !
